plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/MapEditorPart.java

```diff
@@ -28,8 +28,6 @@
 
     Composite getComposite();
 
-    void setDirty(boolean b);
```

Comment:
For now I‘d leave is/setDirty in MapEditorPart interface since it is editor related, isn‘t it.

Reply:
A view can be dirty too, right?